README.md

```diff
@@ -156,54 +141,80 @@ BPANEL_TLS_CERT=<path to cert>
+
+- `securityc` - TLS terminating reverse proxy
+- `bpanel` - bPanel routing + static file server (connecting to the regtest node)
+- `bcoin` - bcoin bitcoin node/wallet (running a regtest node)
+
+(Some plugins require TLS to function properly e.g. for hardware signing
+support).
+
 The `bpanel` service is an http server that acts as a static file server and as a request router
-to backend services as well as a webpack process for building your js files.
+to backend services as well as a webpack process for building your JavaScript files.
 The `bcoin` service is an instance of `bcoin` that supports an http
-server, a wallet server and a bitcoin p2p server.
+server, a wallet + [bmultisig](https://github.com/bcoin-org/bmultisig) server and a bitcoin p2p server.
 
 The `securityc` service generates TLS keys and certs and runs a TLS terminating reverse proxy.
 You can use custom configs to connect to an existing node,
 or use the bcoin docker service to spin up a bcoin node that the webapp will connect to.
 
+Learn more about the architecture of bPanel and related services [at our website](https://bpanel.org/docs/bpanel-overview.html#architecture).
+
 #### Configuration between Docker services
+
 These instructions are for if you want to run bPanel within the `bpanel` service and have it talk to
-a bcoin node running in a container from the `bcoin` service. For example, this is how bPanel works
+a bcoin node running in a container from the `bcoin` service. This is how, for example, bPanel works
 out of the box if you simply run `docker-compose up -d`.
 
 Configurations are shared between the two docker containers using a shared
 docker volume called `configs`. Settings for the bcoin nodes in docker
 are set using environment variables, either in docker-compose.yml or
-an env file (by default in `/etc/bcoin.env` but you can point to whichever and
+an env file (by default in `./etc/regtest.bcoin.env` but you can point to whichever and
 as many env files as you want using the `env_file` configuration in the bcoin
 service). The bcoin node is started with the `bcoin-init.js` script. During this
-process, api keys are generated and all required configurations are saved in a config
+process, API keys are generated and all required configurations are saved in a config
 file called `_docker.conf` in the shared volume.
 
-If the configs volume is mounted to your host machine, you can connect a local
+#### Local bPanel Dev (non-docker) with bcoin docker service
+
+If the configs volume is mounted and mapped to your host machine, you can connect a local
 version of bPanel to it by pointing the `client-id` config at `_docker` and it will
 use the appropriate configs.
 
+For local development, you run just the bcoin docker container (`docker-compose up -d bcoin`)
+and then `npm run start:dev -- --client-id=_docker` (or `npm run start:poll -- --client-id=_docker`
+for Mac since webpack's watch can behave strangely on macOS) to run bPanel and its server
```

Comment:
`since webpack's watch can behave strangely on macOS` maybe reword this sentence

```
(or `npm run start:poll -- --client-id=_docker`
on Mac to enbable webpack's watch with filesystem polling)
```